cmd/admin-config-set.go

```diff
@@ -81,37 +88,55 @@ func (u configSetMessage) JSON() string {
 }
 
 // checkAdminConfigSetSyntax - validate all the passed arguments
-func checkAdminConfigSetSyntax(ctx *cli.Context) {
-	if len(ctx.Args()) == 0 || len(ctx.Args()) > 2 {
+func checkAdminConfigSetSyntax(ctx *cli.Context) string {
+	if len(ctx.Args()) == 0 {
 		cli.ShowCommandHelpAndExit(ctx, "set", 1) // last argument is exit code
 	}
+	if len(ctx.Args()) == 1 {
+		return "fullSet"
+	}
+	// if 2 or more arguments are passed
+	return "partialSet"
 }
 
 // main config set function
 func mainAdminConfigSet(ctx *cli.Context) error {
+	// Initializations
+	aliasedURL := ctx.Args().Get(0)
+	isFullConfigSet := false
 
 	// Check command arguments
-	checkAdminConfigSetSyntax(ctx)
+	if checkAdminConfigSetSyntax(ctx) == "fullSet" {
```

Comment:
Suggestion: let's make the check the first thing to do in this function

Reply:
OK